tests/test_transformation/test_arcsin_transformer.py

```diff
@@ -0,0 +1,74 @@
+import pandas as pd
+import pytest
+from sklearn.exceptions import NotFittedError
+
+from feature_engine.transformation import ArcsinTransformer
+
+
+def test_automatically_find_variables(df_vartypes):
+    # test case 1: automatically select variables
+    transformer = ArcsinTransformer(variables=None)
+
+    # test fails here because of the df_vartypes data that are passed raise ValueError 
+    X = transformer.fit_transform(df_vartypes)
+
+    # expected output
+    transf_df = df_vartypes.copy()
+    transf_df["Age"] = [0.05, 0.047619, 0.0526316, 0.0555556]
+    transf_df["Marks"] = [1.11111, 1.25, 1.42857, 1.66667]
+
+    # test init params
+    assert transformer.variables is None
+    # test fit attr
+    assert transformer.variables_ == ["Age", "Marks"]
+    assert transformer.n_features_in_ == 5
```

Comment:
we also have a test for this parameter: https://github.com/feature-engine/feature_engine/blob/main/tests/estimator_checks/fit_functionality_checks.py#L9

Reply:
I excluded from test_arcsin_transformer.py the tests that can be run adding ArcsinTransformer class to the test_check_estimator_trasformer.py file. When I run pytest on test_check_estimator_transformer.py two tests fails because a Value Error is raised (the data frame contains value outside the [-1, +1] range). How can I handle the situation?

Reply:
hard to say without seeing the code. 

The LogTransformer() should also fail those tests, so I would say, ensure that the tags method is identical to the one from logtransformer? 

It looks like it already is:

https://github.com/feature-engine/feature_engine/pull/444/files#diff-5fd2f94bd07e6b13876da62adb45bfb5a368f3243279765f52ef1e69d608a878R154-R168

Which tests are failing? not any of those?

Reply:
Now the pull request is in draft mode and I have made the changes we discussed locally. Should I make a new pull request or reopen in some way this one? The command to execute on the command line is always the same? (git push origin local_branch_name)?

Reply:
I'm saying this so you can see all the modification and see what's wrong.

Reply:
If you worked on the same branch locally (`arcsin_transformation`) and you push your changes to the same branch so `git push origin arcsin_transformation` that would add the code changes here automatically.

Reply:
Done it. Let me what's wrong.